torch/csrc/distributed/c10d/reducer.h

```diff
@@ -135,7 +146,7 @@ class Reducer {
   // and parameter indices order may change more frequently.
   // For find_unused_parameters = false case, buckets are only rebuilt once,
   // the performance cost is negligible.
-  std::vector<std::vector<size_t>> rebuildBuckets();
+  void rebuildBuckets();
```

Comment:
nit: I noticed that all other methods are using snake names, shall we use `rebuild_buckets` as name here as well?